library/libraries.js

```diff
@@ -1919,5 +1919,30 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         }
         return false;
       }
+    },
+    'WP Rocket':{
+        id:'wp-rocket',
+        icon:'wp-rocket',
+        url: 'https://wp-rocket.me/',
+        npm: null,
+        test: async function (win) {
+            const wpRocketLazyLoad = typeof RocketLazyLoadScripts !== 'undefined'|| typeof RocketPreloadLinksConfig !== 'undefined' ||typeof rocket_lazy !== 'undefined';
```

Comment:
```suggestion
            const wpRocketLazyLoad = typeof RocketLazyLoadScripts !== 'undefined' || typeof RocketPreloadLinksConfig !== 'undefined' || typeof rocket_lazy !== 'undefined';
```

Reply:
Should we remove `rocket_lazy` from this check? That seems like something that could be instantiated as a false positive (but correct me if you think I'm wrong :))

Reply:
@housseindjirdeh 

> Are there any other signals that can be used instead (maybe reading the DOM to see if there's a wp-rocket/assets/css/wpr-admin.min.css stylesheet)?

this CSS file is only used on the WordPress admin page (/wp-admin)
but if the CSS file exists that means that wp-rocket is installed on this WordPress website.
 
> Could we remove this and just rely on the earlier checks here or is that not sufficient?

the other checks are for features in the plugin itself that can be disabled or enabled by the user

RocketLazyLoadScripts                 --> if **Delay JavaScript execution** feature is activated by the user
RocketPreloadLinksConfig           --> if **Preload Links** feature is activated by the user
rocket_lazy                                      --> if **LazyLoad ** feature is activated by the user
style#wpr-usedcss (dom check)  --> if  **Remove Unused CSS** feature is activated by the user

I am doing the fetch for the CSS file as a last resort if the user is not using any of the above features.

For example, if a user is using only minify CSS feature only the fetch check will pass.

Reply:
Thanks for the details. Can we remove the fetch for the CSS file since it's a last resort? That would ensure that the extension isn't making a fetch call on every page and I think the other checks you have are sufficient (but let me know if otherwise) :)

Reply:
@connorjclark @housseindjirdeh 
I removed the fetch call check